src/main/java/com/minecolonies/api/items/ModItems.java

```diff
@@ -10,108 +28,166 @@
+
+    // Hot Dry Biomes
+    // Tier 2
+    public static final DeferredItem<ItemFood> pepper_hummus = custom("pepper_hummus", prop -> new ItemFood(prop.food(new FoodProperties.Builder().nutrition(6).saturationModifier(1.0F).build()), ModJobs.CHEF_ID.getPath(), 2));
+    // Tier 3
+    public static final DeferredItem<ItemFood> pita_hummus = custom("pita_hummus", prop -> new ItemFood(prop.food(new FoodProperties.Builder().nutrition(8).saturationModifier(1.2F).build()), ModJobs.CHEF_ID.getPath(), 3));
+
+    // Require trading
+    // Tier 2
+    public static final DeferredItem<ItemFood> congee = custom("congee", prop -> new ItemFood(prop.food(new FoodProperties.Builder().nutrition(6).saturationModifier(1.0F).usingConvertsTo(Items.BOWL).build()), ModJobs.CHEF_ID.getPath(), 2));
+    // Tier 3
+    public static final DeferredItem<ItemFood> stew_trencher = custom("stew_trencher", prop -> new ItemFood(prop.food(new FoodProperties.Builder().nutrition(8).saturationModifier(1.2F).build()), ModJobs.CHEF_ID.getPath(), 3));
+    public static final DeferredItem<ItemFood> stuffed_pepper = custom("stuffed_pepper", prop -> new ItemFood(prop.food(new FoodProperties.Builder().nutrition(8).saturationModifier(1.2F).build()), ModJobs.CHEF_ID.getPath(), 3));
+
+    // Just dough
+    public static final DeferredItem<Item> muffin_dough = simple("muffin_dough");
+    public static final DeferredItem<Item> manchet_dough = simple("manchet_dough");
+    public static final DeferredItem<Item> raw_noodle = simple("raw_noodle");
+    public static final DeferredItem<Item> butter = simple("butter");
+
+    public static final DeferredItem<ItemLargeBottle> large_empty_bottle = custom("large_empty_bottle", ItemLargeBottle::new);
+    public static final DeferredItem<ItemLargeBottle> large_milk_bottle = custom("large_milk_bottle", prop -> new ItemLargeBottle(prop.craftRemainder(large_empty_bottle.get())));
+    public static final DeferredItem<ItemLargeBottle> large_water_bottle = custom("large_water_bottle", prop -> new ItemLargeBottle(prop.craftRemainder(large_empty_bottle.get())));
+    public static final DeferredItem<ItemLargeBottle> large_soy_milk_bottle = custom("large_soy_milk_bottle", prop -> new ItemLargeBottle(prop.craftRemainder(large_empty_bottle.get())));
+
+    static
+    {
+        /**
+         * Spawn egg colors.
+         */
+        final int PRIMARY_COLOR_BARBARIAN   = 5;
+        final int SECONDARY_COLOR_BARBARIAN = 700;
+        final int PRIMARY_COLOR_PIRATE      = 7;
+        final int SECONDARY_COLOR_PIRATE    = 600;
+        final int PRIMARY_COLOR_MERC        = 8;
+        final int SECONDARY_COLOR_MERC      = 300;
+        final int PRIMARY_COLOR_EG          = 10;
+        final int SECONDARY_COLOR_EG        = 400;
+
+        spawnEgg("barbarianegg", () -> ModEntities.BARBARIAN, PRIMARY_COLOR_BARBARIAN, SECONDARY_COLOR_BARBARIAN);
```

Comment:
Not too long ago people were asking if spawn eggs were still worth it to even have at all, considering raiders can't appear outside of a raid, making these spawn eggs virtually unusable

Reply:
we can hide them from tabs etc, and leave them as command only thing?

Reply:
I mean even if you spawn them in and use it, they will fail

Reply:
I think we can get rid of all eggs here, yes

Reply:
Fine then, should i also delete the registry helper or you want to keep it? i wont have any usage after this tho